nuxeo-features/nuxeo-platform-imaging/nuxeo-platform-imaging-core/src/test/java/org/nuxeo/ecm/platform/picture/core/test/TestPictureRenditions.java

```diff
@@ -112,4 +127,36 @@ public void shouldExposeOnlyMarkedPictureViewsAsRenditions() throws Exception {
         runtimeHarness.undeployContrib("org.nuxeo.ecm.platform.picture.core",
                 "OSGI-INF/imaging-picture-renditions-override.xml");
     }
+
+    @Test
+    public void shouldDeclareRenditionDefinitionImageToPDF() {
+        List<RenditionDefinition> renditionDefinitions = renditionService.getDeclaredRenditionDefinitions();
+        List<RenditionDefinition> imageToPDFRenditionDefinitions = renditionDefinitions.stream()
+            .filter(rD -> rD.getName().equals("imageToPDF")).collect(Collectors.toList());
+        assertEquals(1, imageToPDFRenditionDefinitions.size());
+        RenditionDefinition imageToPDFRenditionDefinition = imageToPDFRenditionDefinitions.get(0);
+        assertEquals("Image.Blob.ConvertToPDF", imageToPDFRenditionDefinition.getOperationChain());
+        assertEquals(1, imageToPDFRenditionDefinition.getFilterIds().size());
+        assertEquals("hasPictureFacet", imageToPDFRenditionDefinition.getFilterIds().get(0));
+    }
+
+    @Test
+    public void shouldMakeRenditionAvailableImageToPDF() throws Exception {
+        Blob source = Blobs.createBlob(FileUtils.getResourceFileFromContext("images/test.jpg"));
+        DocumentModel doc = session.createDocumentModel("/", "picture", "Picture");
+        doc.setProperty("file", "filename", source.getFilename());
+        doc.setProperty("file", "content", source);
+
+        Rendition imageToPDFRendition = renditionService.getRendition(doc, "imageToPDF");
+        assertNotNull(imageToPDFRendition);
+        Blob pdfRenditionFromService = imagingService.convertToPDF(source);
```

Comment:
Why don't you just do `Blob pdfBlob = imageToPDFRendition.getBlob()`?
You don't even need the last test as the rendition already use automation and the `Image.Blob.ConvertToPDF` chain.